### PR TITLE
Add Disqus Support

### PR DESCRIPTION
**Pull request by Kikobeats**
# Setup

For enable disqus, need to add the correctly metadada at `docpress.json`:

```json
"disqus": {
    "shortname": "kikobeats.com",
    "exclude": "index|bibliography|resources"
  }
```

where:

- shortname: Your disqus shortname for load your comments.
- exclude: Regex pattern for exclude disqus for determinate pages. by default is `index`

### CODE REVIEW

**Comment by Kikobeats** [comment]
@rstacruz what do you think? in the future this will be a separate plugin as well 😄

File: index.js [hunk 1]
```diff
@@ -96,15 +99,15 @@ function addAsset (list, what, files, item) {
 function addCss (files, ms, done) {
   const callback = (err, contents) => {
     if (err) return done(err)
-    files['assets/style.css'] = { contents }
+    files['assets/style.css'] = { contents}
```
Comment (Kikobeats) [inline index.js]: ops, this was standard-format automatically

File: lib/helpers/noop.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function noop () {
+}
+
+module.exports = noop
```
Comment (rstacruz) [inline lib/helpers/noop.js]: lol

**Comment by rstacruz** [comment]
Yes, definitely plugin-ify this later.

**Pull request merged**